eclair-core/src/main/scala/fr/acinq/eclair/channel/ChannelEvents.scala

```diff
@@ -53,7 +53,9 @@ case class ChannelErrorOccurred(channel: ActorRef, channelId: ByteVector32, remo
 case class NetworkFeePaid(channel: ActorRef, remoteNodeId: PublicKey, channelId: ByteVector32, tx: Transaction, fee: Satoshi, txType: String) extends ChannelEvent
 
 // NB: this event is only sent when the channel is available
-case class AvailableBalanceChanged(channel: ActorRef, channelId: ByteVector32, shortChannelId: ShortChannelId, localBalance: MilliSatoshi, commitments: Commitments) extends ChannelEvent
+case class AvailableBalanceChanged(channel: ActorRef, channelId: ByteVector32, shortChannelId: ShortChannelId, commitments: Commitments) extends ChannelEvent {
+  def localBalance: MilliSatoshi = commitments.remoteNextCommitInfo.left.toOption.map(_.nextRemoteCommit).getOrElse(commitments.remoteCommit).spec.toRemote
```

Comment:
I would completely remove this `localBalance` field and the corresponding DB table (which is probably very noisy).